src/constants/gameplay.asm

```diff
@@ -108,10 +108,19 @@ DIALOG_CHOICE              equ $0D ; press A to choose
 DIALOG_CLOSING_1           equ $0E
 DIALOG_CLOSING_2           equ $0F
 
+DIALOG_BOX_BOTTOM_FLAG     equ $80 ; display dialog on the bottom
+DIALOG_BOX_BOTTOM_BIT      equ 7
```

Comment:
The usage of `DIALOG_BOX_BOTTOM_FLAG` is not clear to me. It's a specific bit of `wDialogState`, that specified that the dialog is paused, waiting for the A input to display the next lines? Maybe `DIALOG_WAITING_FLAG` could be a possible name?

Reply:
@kemenaran No, `DIALOG_BOX_BOTTOM_FLAG` is a flag in `wDialogState` saying whether the dialog box should be displayed at the bottom of the screen instead of at the top, used when Link is in the upper half of the screen (so the box doesn't obscure him). It's the same as `DIALOG_BOX_BOTTOM_BIT`, just that the former is used as a mask and the latter is used with `set`/`res` opcodes. I'm sure this could be clearer though.